### DISCUSSION TITLE
Sorry dumb question...

### DISCUSSION THREAD

**Discussion by john412smith** [iptv-org/community#407] [Q&A] [answered]
I noticed few months back this repo was removed. Now I see its up again, but under new management? Or is it same management?

**Comment by BellezaEmporium** [comment, marked as answer]
Reopened, without GH Actions, same management.

**Reply by john412smith** [reply]
Well that is crap. If you can find a decent price dedicated that would work for your needs, I'd be willing to fund it if the price is good. I love this project and I am willing to put my money where my mouth is, so to speak.
